src/applications/ezr/components/FormAlerts/SubmissionErrorAlert.jsx

```diff
@@ -12,8 +12,13 @@ const SubmissionErrorAlert = () => {
       <va-alert status="error" uswds>
         <h3 slot="headline">{content['alert-submission-title']}</h3>
         <div>
-          <p>{content['alert-submission-message']}</p>
+          <p>{content['alert-submission-primary-message']}</p>
+          <p>{content['alert-submission-secondary-message']}</p>
         </div>
+        <a href="/health-care/update-health-information/#how-do-i-update-my-information">
+          Learn more about how to update your information by mail, phone, or in
```

Comment:
Should this text be in the `content.json` file too?

Reply:
I'm not sure we want any of this in `content.json`. The pattern I've been following is that `content.json` is for use when we need a title or description inside a `uiSchema`. Anything inside of `components` doesn't need to go inside of `content.json`. At least that's the pattern I've been following.

Reply:
Sorry disregard just realizing this isn't a new change to the component, just an update to the copy. Looks good to me!